hack/golangci_lint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# check for golangci-lint in *.go files
+
+check=0
+
+test_golangci() {
+        golangci-lint run ./...
+        if [ $? -eq 0 ]; then
+                check=1
+        fi
+}	
+
+golangci-lint --version 
+
+if [ $? -eq 0 ]; then
```

Comment:
What is this test for?

Reply:
just to confirm local system has golangci. then no need to install.

Reply:
We shouldn't be using a binary on the system, much less installing a binary somewhere else in the system. If possible we should always download the binary to a location within the repo and call that.

Reply:
fixed